utils/download_test_db.py

```diff
@@ -121,31 +121,88 @@ def download_dataset(dataset):
             # Just a newline.
             print()
 
+def download_caffe2_models ():
+    models = ['densenet121',
+              'inception_v1', 
+              'inception_v2',
+              'lenet_mnist',
+              'resnet50',
+              'shufflenet',
+              'squeezenet',
+              'vgg19', 
+              'zfnet512',
+              'bvlc_alexnet',
+              'en2gr',
+              'quant_resnet50']
+    for modelname in models:
+        for filename in ['predict_net.pbtxt', 'predict_net.pb', 'init_net.pb']:
+            os.system ('wget -nc http://fb-glow-assets.s3.amazonaws.com/models/{}/{} -P  {}'.format(modelname, filename, modelname))
+    
+    for filename in ['dst_dictionary.txt', 'src_dictionary.txt']:
+        os.system ('wget -nc http://fb-glow-assets.s3.amazonaws.com/models/en2gr/{} -P en2gr'.format(filename))
+    return
+
+def download_onnx_models ():
+    for modelname in ['resnet50', 'vgg19', 'squeezenet', 'zfnet512', 'densenet121', 'shufflenet']:
+        os.system ('wget -nc https://s3.amazonaws.com/download.onnx/models/opset_6/{}.tar.gz'.format(modelname))
+        os.system ('tar -xzvf {}.tar.gz'.format(modelname))
+
+    for modelname in ['inception_v1', 'inception_v2', 'bvlc_alexnet']:
+        os.system ('wget -nc https://s3.amazonaws.com/download.onnx/models/opset_8/{}.tar.gz'.format(modelname))
+        os.system ('tar -xzvf {}.tar.gz'.format(modelname))
+
+    for modelname in ['lenet_mnist']:
+        os.system ('wget -nc http://fb-glow-assets.s3.amazonaws.com/models/{}.tar.gz'.format(modelname))
+        os.system ('tar -xzvf {}.tar.gz'.format(modelname))
```

Comment:
Use the `tarfile` module instead of untarring via `os.system`. We also have an `untar` function defined above that might be useful.